lib/Ix/DBIC/Result.pm

```diff
@@ -183,8 +183,8 @@ sub ix_add_properties ($class, @pairs) {
 
 my %DEFAULT_VALIDATOR = (
   integer => Ix::Validators::integer(),
-  string  => Ix::Validators::simplestr(),
-  istring => Ix::Validators::simplestr(),
+  string  => Ix::Validators::string({ oneline => 1 }),
+  istring => Ix::Validators::string({ oneline => 1 }),
```

Comment:
simplestr made sure our strings weren't whitespace only. We don't appear to do that anymore. Is that intentional?

Reply:
Looks good minus one q

Reply:
`string()` has the `printable` option on by default.  `printable` requires the string match `/\pL\pN\pP\pS/`, which will exclude whitespace-only characters.  You can see the test for config `{ nonempty => 1 }` working like this.